src/routes/Profile.jsx

```diff
@@ -0,0 +1,162 @@
+// React Imports
+import React, { useContext, useEffect, useState } from 'react';
+import { useLocation } from 'react-router-dom';
+// Inrupt Imports
+import { useSession } from '@inrupt/solid-ui-react';
+// Material UI Imports
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import CheckIcon from '@mui/icons-material/Check';
+import ClearIcon from '@mui/icons-material/Clear';
+import EditIcon from '@mui/icons-material/Edit';
+import Link from '@mui/material/Link';
+import Typography from '@mui/material/Typography';
+// Contexts Imports
+import { SignedInUserContext } from '../contexts';
+// Component Inputs
+import { ProfileInputField, ProfileImageField } from '../components/Profile';
+
+/**
+ * Profile Page - Page that displays the user's profile card information and
+ * allow users to edit/update them on PASS
+ *
+ * @memberof Pages
+ * @name Profile
+ * @returns {React.JSX.Element} The Profile Page
+ */
+const Profile = () => {
+  const location = useLocation();
+  const { session } = useSession();
+  const { updateProfileInfo, setProfileData, profileData, fetchProfileInfo } =
+    useContext(SignedInUserContext);
+
+  localStorage.setItem('restorePath', location.pathname);
+
+  const [profileName, setProfileName] = useState(profileData?.profileInfo.profileName);
+  const [nickname, setNickname] = useState(profileData?.profileInfo.nickname);
+
+  const [edit, setEdit] = useState(false);
+
+  const loadProfileData = async () => {
+    const profileDataSolid = await fetchProfileInfo(session);
+    localStorage.setItem('restoreProfileData', JSON.stringify(profileDataSolid));
```

Comment:
You don't ever read this item. You can remove it.

Reply:
Unused item in localStorage is removed.